lib/plugins/aws/package/compile/events/stream/index.js

```diff
@@ -12,6 +12,35 @@ class AwsCompileStreamEvents {
     };
   }
 
+  isValidStackImport(variable) {
+    if (Object.keys(variable).length !== 1) {
+      return false;
+    }
+    if (
+      variable['Fn::ImportValue'] &&
+      (variable['Fn::ImportValue']['Fn::GetAtt'] || variable['Fn::ImportValue'].Ref)
+    ) {
+      return false;
+    }
+    const intrinsicFunctions = ['Fn::ImportValue', 'Ref', 'Fn::GetAtt', 'Fn::Sub', 'Fn::Join'];
+    return intrinsicFunctions.some(func => _.has(variable, func));
```

Comment:
Let's rename `func` into `cfInstructionName`, and do `var[cfInstructionName]` instead of `_.has`